### PR TITLE
Iam integrations

### PR DESCRIPTION
**Pull request by dantavori**
## Status
Ready

## Description
A documentation about IAM integrations - what are they, what are they consist of, how to develop.

### CODE REVIEW

**Comment by glicht** [comment]
@dantavori build is failing. Can you try merging from master. I think it is something we fixed.

**Comment by fvigo** [comment]
@dantavori can you add a reference to this in the `sidebars.js` file? I think the proper section is under Integration Advanced Topics ( https://github.com/demisto/content-docs/blob/48f8984d2e0b2e95239f4420f1a3d1b020000443/sidebars.js#L135 )

**Review by yaron-libman — APPROVED** [review]

**Comment by glicht** [comment]
@dantavori status?

**Comment by dantavori** [comment]
@glicht can I merge?

**Comment by glicht** [comment]
> @glicht can I merge?


I think @anara123 wants to review.

File: docs/integrations/iam-integrations.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+       "givenname": "test",
+       "surname": "test"
+   }
+ }
+}
+```
+
+**How to implement**
+
+Complete the **client.create_user()** method, which should retrieve an [IAMUserAppData](#iamuserappdata) object that contains the data of the user in the application.
```
Comment (anara123) [inline docs/integrations/iam-integrations.md:80]: add link to implementation examples for each command.

File: docs/integrations/iam-integrations.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+**How to implement**
+
+Complete the **client.get_user()** method, which should retrieve an [IAMUserAppData](#iamuserappdata) object that contains the data of the user in the application.
+
+
+### **iam-update-user**
+
+**Description**
+
+Updates a user. If the user exists but is disabled, it enables the user.
```
Comment (anara123) [inline docs/integrations/iam-integrations.md:136]: And if the user not exists, then it creates it, no?

File: docs/integrations/iam-integrations.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+**How to implement**
+
+Complete the **client.update_user()** and **client.enable_user()** methods, each should retrieve an [IAMUserAppData](#iamuserappdata) object that contains the data of the user in the application.
+
+
+### **iam-disable-user**
+
+**Description**
+
+Disables a user.
```
Comment (anara123) [inline docs/integrations/iam-integrations.md:187]: It disables/removes , no?

File: docs/integrations/iam-integrations.md [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+
+### **get-mapping-fields**
+
+**Description**
+
+Retrieves a User Profile schema which holds all of the user fields within the application. Used for outgoing-mapping through the Get Schema option. 
+
+**How to implement**
+
+Complete the **client.get_app_fields()** method, which should retrieve a dictionary of fields and their descriptions.
```
Comment (anara123) [inline docs/integrations/iam-integrations.md:240]: add example code or link to each command that we request to implement

**Review by anara123 — APPROVED** [review]
See my comments

**Pull request merged**